src/plugins/curator/components/widget-screen/DataSourceTab.js

```diff
@@ -66,7 +66,6 @@ export default function DataSourceTab(props) {
             label="Content Type"
             options={options}
             isClearable={false}
```

Comment:
@rsumilang that field is required by schema though, in what scenario could/is this blank?

Reply:
@gdbrown it is not required - https://github.com/triniti/schemas/blob/563218e8cdd37309d39b606b33a1d197f94bd9c6/schemas/triniti/curator/mixin/widget-has-search-request/latest.xml#L18-L22

we've got it up and running where you can make an alert widget with no search request (because its not required) but then once made, if you try to edit any field it starts yelling at you for not having set a search_request

Reply:
@gdbrown its blank if you use the create widget modal to create a new widget.

Reply:
@rsumilang @efjacobson oic, blank on the parent object potentially.  putrid.  i'll roll it into the sausage.